test/config-next/wfe.json

```diff
@@ -14,13 +14,16 @@
     "acceptRevocationReason": true,
     "allowAuthzDeactivation": true,
     "debugAddr": "localhost:8000",
+    "raService": {
+      "serverAddresses": ["boulder:9094"],
+      "serverIssuerPath": "test/grpc-creds/minica.pem",
+      "clientCertificatePath": "test/grpc-creds/boulder-client/cert.pem",
+      "clientKeyPath": "test/grpc-creds/boulder-client/key.pem",
+      "timeout": "90s"
```

Comment:
Why the different timeout here? Used to be 15s, which seems reasonable for testing.